@3rdweb-sdk/react/hooks/usePayments.ts

```diff
@@ -862,3 +889,181 @@ export function usePaymentsSellerByAccountId(accountId: string) {
     { enabled: !!paymentsSellerId && !!address },
   );
 }
+
+export type PaymentsWebhooksType = {
+  id: string;
+  sellerId: string;
+  url: string;
+  isProduction: boolean;
+  createdAt: Date;
+};
+
+export function usePaymentsWebhooksByAccountId(accountId: string) {
+  invariant(accountId, "accountId is required");
+
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+  const [getWebhooksBySellerId] = useWebhooksBySellerIdLazyQuery();
+
+  const queryInfo = useQuery(
+    paymentsKeys.webhooks(accountId),
+    async () => {
+      invariant(paymentsSellerId, "no payments seller id found");
+      const { data, error } = await getWebhooksBySellerId({
+        variables: {
+          sellerId: paymentsSellerId,
+        } as WebhooksBySellerIdQueryVariables,
+      });
+
+      if (error) {
+        console.error(error);
+      }
+
+      if (data && data.webhook.length > 0) {
```

Comment:
can be improved a bit if you do a ternary like I do in other places in the component, but that's a nit

`return data && data?.webhook.length > 0 ? data.webhook.map(... : [] as PaymentsWebhookType[]`